pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -726,6 +730,29 @@ def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
                     RemoveRedundancies(),
                 ]
             )
+        else:
+            passlist.extend(
+                [
+                    RemoveBarriers(),
+                    AutoRebase({OpType.CX, OpType.Rz, OpType.H}),
```

Comment:
I think we should allow more gates that `GreedyPauliSimp` can handle so the rebase can run faster. Also easier for the Pauli graph conversion.

Reply:
done